Tests/SwiftLintFrameworkTests/VerticalWhitespaceClosingBracesRuleTests.swift

```diff
@@ -0,0 +1,75 @@
+@testable import SwiftLintFramework
+import XCTest
+
+class VerticalWhitespaceClosingBracesRuleTests: XCTestCase {
+    func testVerticalWhitespaceClosingBracesWithDefaultConfiguration() {
```

Comment:
Please remove this test. The examples are already verified in `GeneratedTests.swift`.

Reply:
Removed.